components/clp-py-utils/clp_py_utils/start-spider-worker.py

```
@@ -0,0 +1,46 @@
+    args = parse_args()
+    concurrency = args.concurrency
+    storage_url = args.storage_url
+    host = args.host
+
+    clp_home = os.getenv("CLP_HOME", "/opt/clp")
+    spider_worker_path = pathlib.Path(clp_home) / "bin" / "spider_worker"
+
+    # Start multiple spider workers
+    processes = []
+    for _ in range(concurrency):
+        process = subprocess.Popen(
```

Ruff (0.14.4)

`subprocess` call: check for execution of untrusted input

(S603)